src/webob/acceptparse.py

```diff
@@ -408,6 +408,24 @@ def generator(value):
                 )
         return generator(value=value)
 
+    def _parse_and_normalize_offers(self, offers):
+        """
+        Throw out any offers that do not match the media type ABNF.
+
+        :return: A list of ABNF-validated offers split into the format:
+                 ``[offer_index, offer_type_subtype, offer_media_params]``.
+
+        """
+        lowercased_offers_parsed = []
+        for index, offer in enumerate(offers):
+            match = self.media_type_compiled_re.match(offer.lower())
+            # The regex here is only used for parsing, and not intended to
+            # validate the offer
+            if not match:
+                continue
+            lowercased_offers_parsed.append([index] + list(match.groups()))
```

Comment:
I think the above 3 lines can be replaced with

    if match:
        lowercased_offers_parsed.append([index] + list(match.groups()))

Reply:
fixed